packages/components/src/InputTime/InputTime.types.ts

```diff
@@ -43,15 +43,17 @@ export interface InputTimeProps
    * Function called when user changes input value.
    */
   onChange?(newValue?: Date): void;
-  /**
-   * The version of the component.
-   */
-  version?: 1 | 2;
+}
+
+export interface InputTimeLegacyProps extends InputTimeProps {
+  version?: 1;
 }
 
 export interface InputTimeRebuiltProps
   extends Omit<InputTimeProps, "defaultValue" | "version"> {
```

Comment:
So I noticed that the rebuilt props provides hints for validations and `onValidation` which comes from RHF. I think it should be omitted from this as well

Reply:
ah right. good catch.